bundle/Features/ContentOnTheFly.feature

```diff
@@ -1,25 +1,25 @@
 Feature: Basic interactions for ContentOnTheFly
 
   Background:
-    Given I am logged in as admin on StudioUI
+    Given I am logged in as admin on PlatformUI
 
     @javascript @common @contentOnTheFly
       Scenario: I can create Content directly from the Dashboard
       Given I start creating content from the Dashboard
```

Comment:
Nitpick: Indentation is different in first scenario compared to the rest in this file. Also it's different than in other bundles e.g Date Based Publisher (4 & 6 spaces vs. 4 & 8 spaces).